static/defaults.config.php

```diff
@@ -310,10 +310,10 @@
 		// Number of "free" searches when system => permit_crawling is enabled.
 		'free_crawls' => 10,
 
-		// groupedit_image_limit (Integer)
-		// Number of contacts at which the group editor should switch from display the profile pictures of the contacts to only display the names.
+		// circle_edit_image_limit (Integer)
+		// Number of contacts at which the circle editor should switch from display the profile pictures of the contacts to only display the names.
 		// This can alternatively be set on a per-account basis in the pconfig table.
-		'groupedit_image_limit' => 400,
```

Comment:
Is there a backward-compatible option (deprecated of course) still possible? This might break user's setup.

Reply:
Yes, we will check the value of `groupedit_image_limit` first, then `circle_edit_image_limit`.